validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandler.java

```diff
@@ -96,6 +97,31 @@ private ForkInfo mapGetForkResponse(final GetForkResponse response) {
         });
   }
 
+  @Override
+  public SafeFuture<Optional<List<ValidatorDuties>>> getAttestationDuties(
+      final UInt64 epoch, final Collection<Integer> validatorIndexes) {
+    return asyncRunner.runAsync(
+        () -> {
+          final List<ValidatorDuties> validatorDuties =
+              apiClient.getAttestationDuties(epoch, validatorIndexes).stream()
+                  .map(this::mapToApiValidatorDuties)
+                  .collect(Collectors.toList());
+
+          return Optional.of(validatorDuties);
+        });
+  }
+
+  private ValidatorDuties mapToApiValidatorDuties(final AttesterDuty attesterDuty) {
+    return ValidatorDuties.withDuties(
+        attesterDuty.pubkey.asBLSPublicKey(),
+        attesterDuty.validatorIndex.intValue(),
+        attesterDuty.committeeIndex.intValue(),
+        attesterDuty.validatorCommitteeIndex.intValue(),
+        attesterDuty.committeeLength.intValue(),
+        List.of(),
+        attesterDuty.slot);
```

Comment:
I'm pretty sure this isn't called anywhere currently, because the empty list would cause problems...
I'm not overly concerned, but when we switch to using this remote implementation it'll need to be resolved.

Reply:
This would work out better if we split `ValidatorDuties` into separate parts.